responsive-image/responsive-image.css

```diff
@@ -0,0 +1,9 @@
+.img-container {
```

Comment:
Trick for responsive images is `max-width:100%`

Reply:
> 
> 
> Trick for responsive images is `max-width:100%`

I have made this a thumb rule for all of my images in a web page.

Reply:
Thank you for your feedbacks purva. The points you have made have been taken care of!